spring-cloud-stream/src/test/resources/org/springframework/cloud/stream/binder/sink-binding-pubsub-test.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.stream.bindings.input=topic:testpubsub
+spring.cloud.stream.group=tgroup
```

Comment:
I believe the question here is whether the same component can be part of two separate consumer groups, with respect to its different inputs. I'd be tempted to answer 'no' ...